code/datums/martial/cqc.dm

```diff
@@ -196,7 +196,8 @@
 		log_combat(A, D, "knocked out (Chokehold)(CQC)")
 		D.visible_message("<span class='danger'>[A] puts [D] into a chokehold!</span>", \
 							"<span class='userdanger'>[A] puts you into a chokehold!</span>")
-		D.SetSleeping(400)
+		if(D.silent <= 10)
```

Comment:
Not entirely sure what this accomplishes. You can get krav maga gloves and achieve basically this without the fuss. You should consider some damage on top of this. Possibly stamina damage. Though in all honesty you probably would be better off killing someone expediently with your other moves.

Reply:
mute and they're already in a really bad position in a microstun + aggrograb, enough to where you could probably take them out on that, a little bit of stamdamage would probably be good though.